infinispan-remote/infinispan-server-testconfig/wildfly-trimmed-config.xml

```diff
@@ -93,6 +93,9 @@
                 <pattern-formatter pattern="%d{yyyy-MM-dd HH:mm:ss,SSS} %-5p [%c] (%t) %s%e%n"/>
             </formatter>
         </subsystem>
+        <subsystem xmlns="urn:jboss:domain:deployment-scanner:2.0">
+            <deployment-scanner path="deployments" relative-to="jboss.server.base.dir" scan-interval="1" runtime-failure-causes-rollback="${jboss.deployment.scanner.rollback.on.failure:false}"/>
```

Comment:
@DavideD , do we use this configuration in CI? It works for me locally, but somehow fails to rescan deployments in Jenkins.

Reply:
I will check but we don't do anything specific on Jenkins for the Infinispan Server. It just run the build the same way you'd run it locally

Reply:
If this branch works for you I can try it on my machine and let you know

Reply:
Can you access this link?
http://ci.hibernate.org/view/Pull%20Requests/job/hibernate-ogm-PR/ws/infinispan-remote/target/infinispan-server/